tests/python/pants_test/build_graph/test_target_filter_subsystem.py

```diff
@@ -1,60 +0,0 @@
-# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
```

Comment:
This feature has never worked with v2. I think we should probably remove it in a followup? @benjyw?